### PR TITLE
Big Pictures!

### PR DESCRIPTION
**Pull request by onebarloop**
Again a rather small PR: The Tattoos at the Artist-Page are now clickable - a popup appears which shows them much bigger.

### CODE REVIEW

File: components/Artist.tsx [hunk 1]
```diff
@@ -14,33 +15,54 @@ export default function Artist({
 }: ArtistInterface): JSX.Element {
   const { handleLike, likes } = useLikes();
 
+  const [popUp, setPopUp] = useState<string | null>(null);
+
+  // function handlePicture(tattoo: string | null = null) {
+  //   setPopUp(tattoo);
+  // }
```
Comment (AntjeWalter) [inline components/Artist.tsx]: This could be deleted if its not needed anymore :)

File: components/Artist.tsx [hunk 2]
```diff
@@ -14,33 +15,54 @@ export default function Artist({
...
+                <Button
+                  name={likes.includes(_id) ? "Dislike" : "Like"}
+                  onClick={() => handleLike(_id)}
+                />
+              </div>
+            </section>
+            <section>
+              <div>{location.city}</div>
+              <div>{location.streetname + " " + location.number}</div>
+            </section>
```
Comment (AntjeWalter) [inline components/Artist.tsx]: Maybe you could make some divs to p or something else, to have a little less div-salad and a little more accessibility? :)

File: components/Artist.tsx [hunk 3]
```diff
@@ -14,33 +15,54 @@ export default function Artist({
...
+          </StyledInfoBox>
 
-      <StyledSchedule>
-        <DatePick dates={dates} inline />
-      </StyledSchedule>
+          <StyledGalery>
```
Comment (AntjeWalter) [inline components/Artist.tsx]: Gallery with two l ;)

**Review by AntjeWalter — APPROVED** [review]
very nice, just some very small comments :)

File: components/Artist.tsx (hunk 1 above)
Comment (onebarloop) [reply components/Artist.tsx]: > 

Thanks for checking Antje 😁

**Comment by marvinRoellecke** [comment]
Dieser Pull Request wird Ihnen präsentiert von Aiman Abdallah

**Comment by onebarloop** [comment]
> Dieser Pull Request wird Ihnen präsentiert von Aiman Abdallah

Okay der hat gedauert bei mir 😅🤣

**Pull request merged**